src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,362 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+/**
+ * Retrieve an Apple Pay Payment Session object.
+ *
+ * Post a request to Apple Pay server's Payment Session endpoint using two-way TLS.
+ *
+ * Parameters:
+ * - pemCertPath: Path to the cert of the split Merchant Identity certification,
+ * associated with your merchantID needed to make the call.
+ * - keyCertPath: Path to the key cert of the split Merchant Identity certification,
+ * associated with your merchantID needed to make the call.
+ * - keyCertPassword: Key certification password for the key certification.
+ * - validationUrl: Validation URL received from session.onvalidatemerchant on the client.
+ * - merchantIdentifier: Your merchant ID provided by Apple.
```

Comment:
is this the same for every request? maybe it should be on the gateway too. or could you conceivably have multiple?

and how about for applicationUrl below?

Reply:
I think you could have multiple, depends on the context. I'd rather leave them here since they're so specific to the context of the authorize request being sent.

Reply:
ok sounds good